src/RateTargetBaseInterestRateStrategy.sol

```diff
@@ -10,9 +10,7 @@ import {IDefaultInterestRateStrategy} from 'aave-v3-core/contracts/interfaces/ID
 import {IReserveInterestRateStrategy} from 'aave-v3-core/contracts/interfaces/IReserveInterestRateStrategy.sol';
 import {IPoolAddressesProvider} from 'aave-v3-core/contracts/interfaces/IPoolAddressesProvider.sol';
 
-interface IRateSource {
-  function getAPR() external view returns (uint256);
-}
+import {IRateSource} from './interfaces/IRateSource.sol';
```

Comment:
```suggestion
import { IRateSource } from "./interfaces/IRateSource.sol";
```

Reply:
Yeah so this is following the Aave-style of 2 spaces, etc to minimize the diff, but I will be just removing this and merging into the other PR later, so let's ignore these for now.